azerothcore/azerothcore-wotlk
# fix(Core): Quest Question marks

## Changes Proposed:
-  Imported logic from Trinitycore with a few tweeks for Azerothcore

## Issues Addressed:

- Closes part of #7897 
- Closes part of #8151
- Closes https://github.com/chromiecraft/chromiecraft/issues/2535

## SOURCE:

Several quests that are repeatable should have a Blue Question mark. This can be seen in loads of videos on classic wow.
Imported from and adapted https://github.com/TrinityCore/TrinityCore/pull/23338

## Tests Performed:

- Tested on Windows 10

## How to Test the Changes:


1. Follow instructions on #7897 and see that the Dwarf has a blue question mark
2. Follow instructions on https://github.com/chromiecraft/chromiecraft/issues/2535 and check for the blue question mark
3. Follow the instructions on the issue fixed by the Trinitycore PR.
4. .go c id 25580 (Learn fishing on any capital first and check for the blue "!")
5. .go c id 20735 : Should have lvl 80 for this weekly and a yellow exclamation mark.

## Known Issues and TODO List:


- [x] daily must have blue "!" -- 11665
- [x]  weekly must have yellow "!" -- 24583


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
